Detectors/ITSMFT/ITS/workflow/src/TrackerSpec.cxx

```diff
@@ -65,6 +65,16 @@ void TrackerDPL::init(InitContext& ic)
     geom->fillMatrixCache(o2::math_utils::bit2Mask(o2::math_utils::TransformType::T2L, o2::math_utils::TransformType::T2GRot,
                                                    o2::math_utils::TransformType::T2G));
 
+    std::string matLUTPath = ic.options().get<std::string>("material-lut-path");
+    std::string matLUTFile = o2::base::NameConf::getMatLUTFileName(matLUTPath);
+    if (o2::base::NameConf::pathExists(matLUTFile)) {
+      auto* lut = o2::base::MatLayerCylSet::loadFromFile(matLUTFile);
+      o2::base::Propagator::Instance()->setMatLUT(lut);
+      LOG(INFO) << "Loaded material LUT from " << matLUTFile;
+    } else {
+      LOG(INFO) << "Material LUT " << matLUTFile << " file is absent, only TGeo can be used";
+    }
+
```

Comment:
This is ok, but the the LUT is requested and the file is not there, it will anyway switch to TGeo automatically.

Reply:
Do you mean that if the file is not there it will default to TGeom? That sounds reasonable, no? 
Or I misunderstood those lines and that never uses the LUT? I did a check and the pointer should be set, also the log agrees.

Reply:
I mean if LUT is requested but it is not set, then it will automatically use TGeo:
https://github.com/AliceO2Group/AliceO2/blob/fc839b714d6100b07d2e86d8badcd0fb9eca9632/Detectors/Base/src/Propagator.cxx#L542-L544

Reply:
Yes, this was then clear to me. What I don't understand is that we want it to fail if LUT is requested or are we happy with this default.